src/components/Garden/ProposalDetail/ProposalDetail.js

```diff
@@ -171,143 +177,159 @@ function ProposalDetail({
-                              commonPool,
-                              requestToken.decimals
-                            )}
-                          </strong>{' '}
-                          {requestToken.symbol} currently in the common pool.
-                        </span>
-                      ) : (
-                        <span>
-                          This suggestion is for signaling purposes and is not
-                          requesting any {requestToken.symbol}
-                        </span>
-                      )}
-                    </div>
-                  </div>
-                  <div
-                    css={`
-                      display: grid;
-                      grid-template-columns: ${layoutName !== 'small'
-                        ? 'auto auto auto'
-                        : 'auto'};
-                      grid-gap: ${layoutName !== 'small' ? 5 * GU : 2.5 * GU}px;
-                    `}
-                  >
-                    <DataField
-                      label="Forum"
-                      value={
-                        link ? (
-                          <Link href={link} external>
-                            Read the full proposal
-                          </Link>
-                        ) : (
-                          <span
-                            css={`
-                              ${textStyle('body2')};
-                            `}
-                          >
-                            No link provided
-                          </span>
-                        )
-                      }
-                      css="grid-column-start: span 2;"
-                    />
-                    <DataField
-                      label="Status"
-                      value={<ProposalStatus proposal={proposal} />}
-                    />
-                    {fundingProposal && (
-                      <Amount
-                        requestedAmount={requestedAmount}
-                        requestedAmountConverted={requestedAmountConverted}
-                        requestToken={requestToken}
-                        stable={stable}
-                        stableToken={stableToken}
+                      <DataField
+                        label="Status"
+                        value={<ProposalStatus proposal={proposal} />}
                       />
-                    )}
+                      {fundingProposal && (
+                        <Amount
+                          requestedAmount={requestedAmount}
+                          requestedAmountConverted={requestedAmountConverted}
+                          requestToken={requestToken}
+                          stable={stable}
+                          stableToken={stableToken}
+                        />
+                      )}
 
-                    {fundingProposal && (
+                      {fundingProposal && (
+                        <DataField
+                          label="Beneficiary"
```

Comment:
Here too, have some reason to use two IFs here instead the One? Kinda better readibility to know if that component should appear?

Reply:
I could do that of course. But the this should be done with sections, if we need in the future to show the description for example I just need to remove the condition not rearrange all the code.

Reply:
Make sense.